api/managementpb/ia/pagination.proto

```diff
@@ -0,0 +1,21 @@
+syntax = "proto3";
+
+package ia.v1beta1;
+
+option go_package = "api/managementpb/ia;iav1beta1";
+
+// PaginationRequest represents request values for page by page pagination.
+message PaginationRequest {
+  // Maximum number of results per page.
+  int32 page_size = 1;
+  // Index of the requested page, starts from 0.
+  int32 page = 2;
```

Comment:
Typically `cursor` gives a more slick UX than `offset` approach.  Maybe we can think about it?

Reply:
Cursor is better also performance-wise but I don't think it is suitable for us. There won't be so many records >10000.
Also implementation is based on UI and there are pages. With cursor approach we can't go to exact page, just to the next.

Reply:
Yes, my message was about the representation approach, but it's out of code review scope.